filter/cond/filtercond.go

```diff
@@ -69,7 +110,7 @@ func InitHandler(ctx context.Context, raw *config.ConfigRaw) (config.TypeFilterC
 		config.Logger.Warn("filter cond config filters empty, ignored")
 		return &conf, nil
 	}
-	conf.expression, err = govaluate.NewEvaluableExpression(conf.Condition)
+	conf.expression, err = govaluate.NewEvaluableExpressionWithFunctions(conf.Condition, GetBuiltInFunctions())
```

Comment:
I think it will not be initialized unless somebody choose to use it.

Reply:
For dynamic loading purpose, you can init in InitHandler(), but errors should be init at first, because it may be used by other package to catch error and check.
BTW, I think the cost is low for storing functions map to a static variable, just only a few memory usage. If someone need to run the app in memory limited environment, it's better to recompile a smaller binary with necessary modules by pruning modloader package.

Reply:
Yep, you are right. I will make a commit.